src/pages/EntityDetails/Model/Logs/ActionLogs/_action-logs.scss

```diff
@@ -28,6 +28,13 @@
     }
   }
 
+  .entity-details__payload-modal {
+    position: relative;
```

Comment:
It might be good to leave a comment here about the stacking context so we know why this is here in the future.